tasks/main.yml

```diff
@@ -28,6 +52,12 @@
   with_items: '{{ disk_additional_disks }}'
   tags: ['disk']
 
+- name: "Disable periodic fsck on ext3 or ext4 formatted disks"
+  shell: tune2fs -c0 -i0 {{ item.disk }}1
+  with_items: '{{ disk_additional_disks }}'
+  when: "disk_additional_disks and ( item.fstype == 'ext4' or item.fstype == 'ext3' )"
```

Comment:
I applied an option to control the task `disable_periodic_fsck` (boolean) within the array `disk_additional_disks` .
@MiLk It is good enough for you?